src/components/textToSpeech/js/TextToSpeech.js

```diff
@@ -186,6 +214,12 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         };
 
         var toSpeak = new SpeechSynthesisUtterance(text);
+        // Store toSpeak additional on the currentUtterance member to help deal with the issue
+        // with premature garbage collection described at https://bugs.chromium.org/p/chromium/issues/detail?id=509488#c11
+        // this makes the speech synthesis behave much better in Safari in
+        // particular
+        // that.currentUtterance = toSpeak;
```

Comment:
The perils of running 50 differential manual tests at the end of the day is that you do things like this and comment out the actual fix for the problem.